src/Tribe/Data_API.php

```diff
@@ -0,0 +1,517 @@
+<?php
+
+if ( ! class_exists( 'Tribe__Tickets__Data_API' ) ) {
```

Comment:
I totally get that this isn't unique to this class and has been a common practice throughout our codebase ... but in a world where we utilize autoloading, is there any benefit to this? If this class was already defined, the autoloader wouldn't have been called.